mulighetsrommet-api/src/main/resources/db/migration/V134__avtale_arrangor_kontaktperson.sql

```diff
@@ -0,0 +1,11 @@
+create table avtale_arrangor_kontaktperson
+(
+    arrangor_kontaktperson_id uuid not null references arrangor_kontaktperson (id) on delete cascade,
+    avtale_id                 uuid not null references avtale (id) on delete cascade,
+    primary key (arrangor_kontaktperson_id, avtale_id)
+);
+
+insert into avtale_arrangor_kontaktperson (arrangor_kontaktperson_id, avtale_id)
+select arrangor_kontaktperson_id, id from avtale where arrangor_kontaktperson_id is not null;
+
+alter table avtale drop column arrangor_kontaktperson_id;
```

Comment:
Mulig du må kjøre en `drop view if exists avtale_admin_dto_view` for at denne skal funke i dev/prod?

Reply:
Selv om du la tilbake den i R__ filene?

Reply:
Legger den til uansett

Reply:
Ja, siden R__-filene først kjører etter V__-filene :/